txbridge/src/test/java/org/jboss/jbossts/txbridge/tests/outbound/junit/DisabledContextPropagationTests.java

```diff
@@ -209,13 +210,16 @@ public void testNoTransactionWithEnabledJTAOverWSATFeature(@ArquillianResource U
     public void testNonTransactionlServiceWithEnabledJTAOverWSATFeature(@ArquillianResource URL baseURL) throws Exception {
         List<NameValuePair> parameters = new ArrayList<NameValuePair>();
         parameters.add(new BasicNameValuePair("isCommit", "true"));
-        parameters.add(new BasicNameValuePair("isTransaction", "ture"));
+        parameters.add(new BasicNameValuePair("isTransaction", "true"));
         parameters.add(new BasicNameValuePair("isWSATService", "false"));
         parameters.add(new BasicNameValuePair("isJTAOverWSATFeatureEnabled", "true"));
         parameters.add(new BasicNameValuePair("clientType", String.valueOf(TestATClient.CLIENT_WITH_JTA_FEATURE)));
-
-        List<String> invocations = makeRequest(baseURL, parameters);
-        assertInvocations(invocations);
```

Comment:
It would be dead code because exception is thrown here:
makeRequest(baseURL, parameters); 
Otherwise test fails:
Assert.fail("'MustUnderstand headers' exception should be thrown.");